apps/condo/domains/miniapp/schema/B2BAppAccessRightSet.js

```diff
@@ -35,15 +76,112 @@ const B2BAppAccessRightSet = new GQLListSchema('B2BAppAccessRightSet', {
             },
         },
 
+        name: {
+            schemaDoc: 'Name for right set to distinguish it',
+            type: 'Text',
+            isRequired: true,
+            defaultValue: 'default',
```

Comment:
`default` is not matching schemaDoc, if all sets became default = no way to distinguish them :)

Reply:
It does match. We had one B2BAppAccessRightSet on miniapp. Now we have multiple. And to make multiple sets we must have one of type GLOBAL first. Sets, which we already have, becomes GLOBAL aka 'default'